spec-main/extensions-spec.ts

```diff
@@ -184,28 +184,38 @@ describe('chrome extensions', () => {
   });
 
   describe('chrome.runtime', () => {
-    let content: any;
-    before(async () => {
+    let w: BrowserWindow;
+    const exec = async (name: string) => {
+      const p = emittedOnce(ipcMain, 'success');
+      await w.webContents.executeJavaScript(`exec('${name}')`);
+      const [, result] = await p;
+      return result;
+    };
+    beforeEach(async () => {
       const customSession = session.fromPartition(`persist:${require('uuid').v4()}`);
       await customSession.loadExtension(path.join(fixtures, 'extensions', 'chrome-runtime'));
-      const w = new BrowserWindow({ show: false, webPreferences: { session: customSession } });
-      try {
-        w.loadURL(url);
-        await emittedOnce(w.webContents, 'dom-ready');
-        content = JSON.parse(await w.webContents.executeJavaScript('document.documentElement.textContent'));
-        expect(content).to.be.an('object');
-      } finally {
-        w.destroy();
-      }
+      w = new BrowserWindow({ show: false, webPreferences: { session: customSession, nodeIntegration: true } });
+      w.loadURL(url);
+      await emittedOnce(w.webContents, 'dom-ready');
```

Comment:
do we need to wait for `dom-ready` here or can we just `await w.loadURL`?

Reply:
Nope. I've updated extension tests to only await `dom-ready` when necessary.